mesheryctl/pkg/utils/helpers.go

```diff
@@ -257,7 +263,30 @@ func PreReqCheck(subcommand string, focusedContext string) error {
 			}
 		}
 	} else if currCtx.Platform == "kubernetes" {
+		if err != nil {
+			return errors.Wrap(err, "failed to create new client")
+		}
+
+		var kubeconfig string
+		home := homedir.HomeDir()
+		if home != "" {
+			kubeconfig = filepath.Join(home, ".kube", "config")
+		}
 
+		config, err := clientcmd.BuildConfigFromFlags("", kubeconfig)
+		if err != nil {
+			return err
+		}
+
+		clientset, err := kubernetes.NewForConfig(config)
+		if err != nil {
+			return err
+		}
```

Comment:
We can leverage MeshKit for creating the client. This could be replaced with one line-

```
            // create an kubernetes client
			client, err := meshkitkube.New([]byte(""))

			if err != nil {
				return err
			}
```

Could you refer the system status command in the `status.go` file? It has similar implementation...

This would make the code really simple and we can abstract a lot of functions.